### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by iBel251**

### CODE REVIEW

**Review by iBel251 — COMMENTED** [review]
You have been doing a tremendous job so far 👌
There are some issues that you still need to work on.

Required Changes ♻️
Please check the comments below. Thankyou. 👏 👏

**Review by iBel251 — COMMENTED** [review]
Kindly, add a descriptive title for your Pull Request. your Pull Request should have a short summary describing what changes are inside.

**Review by iBel251 — COMMENTED** [review]
Your README looks almost empty of content, but I think you can improve it by making it more professional. A professional README file should have a project title, project description, list of technology used in the project, and instructions to set the project up locally. please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.

File: index.html [hunk 1]
```diff
@@ -7,14 +7,13 @@
...
-
     
       <h1 id="title" style="text-align: center; text-transform: uppercase;">Microverse Survey Form</h1>
       <p style="text-align: center; text-transform: uppercase; " id="description">This Survey form is intented to collect data of all prospective students of Microverse</p>
    
       <div style="margin-top: 4%;"></div>
 
-    <form id="survey-form" action="">
+    <form id="survey-form" action="" >
       <p style="font-weight: bold; text-transform: uppercase;">Enter Your Personal Details</p>
```
Comment (iBel251) [inline index.html:17]: To respect best practices, Please try to use an external CSS file to style your html, using inline styling will make your code complex and hard to maintain.

File: index.html [hunk 2]
```diff
@@ -7,14 +7,13 @@
...
   <body>
-
     
       <h1 id="title" style="text-align: center; text-transform: uppercase;">Microverse Survey Form</h1>
       <p style="text-align: center; text-transform: uppercase; " id="description">This Survey form is intented to collect data of all prospective students of Microverse</p>
    
       <div style="margin-top: 4%;"></div>
 
-    <form id="survey-form" action="">
+    <form id="survey-form" action="" >
```
Comment (iBel251) [inline index.html:16]: Optional - Kindly include an action attribute that will ensure that the form data is submitted to any destination.

**Review by aimemalaika — COMMENTED** [review]
Hi @iBel251, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

## OPTIONAL COMMENT

- [ ] Kindly advise the developer to give a meaningful name to the branches. It is important because by looking at the branch name, you can understand what this Git branch is about and its purpose. Check this link for more information [Git Branching Naming Convention: Best Practices](https://codingsight.com/git-branching-naming-convention-best-practices/).

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**